### PR TITLE
[PACKAGE] - Httpexceptionfilter

### PR DESCRIPTION
**Pull request by diephil**
## Description

Initialize the `@algoan/http-exception-filter` package.
Re-usable tsconfigs
Re-usable jest configs
Testing the exception filter
Updated readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,4 +17,10 @@ See [the documentation here](./packages/nestjs-pagination/README.md).
...
 
-See [the documentation here](./packages/nestjs-logging-interceptor/README.md).
\ No newline at end of file
+See [the documentation here](./packages/nestjs-logging-interceptor/README.md).
+
+## NestJS Http Exception Filter
+
+A simple NestJS Http Exception Filter.
+
+See [the documentation here](./packages/http-exception-filter/README.md).
```
[the remark this answers is not in the document]
Comment (diephil) [reply README.md:26]: Removing the README.md might work on github, but leaving the full link will make work links everywhere (in markdown editors)

[the remark this answers is not in the document]
Comment (ccoeurderoy) [reply README.md:26]: What does matter here, is GitHub 🙂

**Comment by diephil** [comment]
I've tested this filter locally on our algoan nestjs apps successfully (via npm pack).

File: packages/http-exception-filter/README.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+## Installation
+
+```bash
+npm install --save @algoan/http-exception-filter
+```
+
+## Usage
+### Default usage
+Use the interceptor as a global interceptor (cf. refer to the [last paragraph](https://docs.nestjs.com/interceptors#binding-interceptors) of this section for more details).
```
Comment (ccoeurderoy) [inline packages/http-exception-filter/README.md:21]: Actually, it isn't an interceptor, it is an [exception filter](https://docs.nestjs.com/exception-filters).

File: packages/http-exception-filter/README.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  <a href="http://nestjs.com"><img src="https://nestjs.com/img/logo_text.svg" alt="Nest Logo" width="320" /></a>
+</p>
+
+<p align="center">
+  A <a href="https://github.com/nestjs/nest">Nest</a> http exception filter.
+</p>
+
+# NestJS Http Exception Filter
+
+A simple NestJS Http [Exception Filter](https://docs.nestjs.com/exception-filters).
```
Comment (ccoeurderoy) [inline packages/http-exception-filter/README.md:11]:
```suggestion
A simple NestJS Http [Exception Filter](https://docs.nestjs.com/exception-filters) logging the HTTP response and formatting error returned by the API.
```

**Comment by diephil** [comment]
@ccoeurderoy thanks for the review, it should be fixed now.

File: packages/http-exception-filter/README.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+## Installation
+
+```bash
+npm install --save @algoan/http-exception-filter
+```
+
+## Usage
+### Default usage
+Use the http exception filter as a global filter (cf. refer to the [last paragraph](https://docs.nestjs.com/interceptors#binding-interceptors) of this section for more details).
```
Comment (ccoeurderoy) [inline packages/http-exception-filter/README.md:21]:
```suggestion
Use the http exception filter as a global filter (cf. refer to the [last paragraph](https://docs.nestjs.com/exception-filters#binding-filters) of this section for more details).
```

File: packages/http-exception-filter/README.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+  <a href="http://nestjs.com"><img src="https://nestjs.com/img/logo_text.svg" alt="Nest Logo" width="320" /></a>
+</p>
+
+<p align="center">
+  A <a href="https://github.com/nestjs/nest">Nest</a> http exception filter.
+</p>
+
+# NestJS Http Exception Filter
+
+A simple NestJS Http [Exception Filter](https://docs.nestjs.com/exception-filters) logging the HTTP response and formatting error returned by the API.
```
Comment (ccoeurderoy) [inline packages/http-exception-filter/README.md:11]:
```suggestion
A simple NestJS Http [Exception Filter](https://docs.nestjs.com/exception-filters) logging the HTTP response and formatting errors returned by the API.
```

**Pull request merged**